src/compiler/crystal/tools/formatter.cr

```diff
@@ -1530,9 +1530,13 @@ module Crystal
     def visit(node : FunDef)
       write_keyword :fun, " "
 
-      check :IDENT
-      write node.name
-      next_token_skip_space
+      case @token.type
+      when :IDENT, :CONST
```

Comment:
Maybe it would be better to let `check` receive multiple arguments and check for any of them?

Reply:
was thinking about it too, will probably do that

Reply:
Is this needed anywhere inside `formatter.cr`? If not, put it in this PR, if it *is* used in other places, do what those places do and refactor in a new PR.

Reply:
Good idea, it looks like it was the only place where a `case @token.type` was used this way. So I'll leave this change in this PR.